src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package mate.academy.bookstore.dto;
 
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull(message = "can't be null")
     private String title;
```

Comment:
For text fields I would suggest to validate length(size) too. Because you have length restrictions on DB level, so it's better to check it here too.